v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/datastream/ChangeEventConvertorTest.java

```diff
@@ -81,6 +81,9 @@ static Ddl getTestDdl() {
             .column("int64_field")
             .int64()
             .endColumn()
+            .column("float32_field")
```

Comment:
Shouldn't there be changes in v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/datastream/ChangeEventConvertor.java?

Reply:
No, that file only has transformations for primary keys. FLOAT32 is not a primary keyable type, which is also why we do not add it to the list of primary keys below

Reply:
yeah makes sense, i didn't notice that the file code had been refactored